sympy/concrete/summations.py

```diff
@@ -574,13 +576,16 @@ def eval_sum(f, limits):
 
 
 def eval_sum_direct(expr, limits):
+    from sympy.core import Add
```

Comment:
@smichr I'm on a long-standing quest to eliminate C from SymPy entirely.
My reason is that classes that get their members installed at runtime make any static analysis tool go bonkers.
@asmeurer also said he'd be wanting to get rid of it for a long time, but I haven't read his full list of reasons.

@vramana we wanted to go from absolute to relative imports (mostly so that people can put sympy into a submodule instead of forcing them to make it a top-level module).
I think we ran into trouble with relative imports that go up a level (e.g. ```import ..special.bessel```), and didn't complete the change.

Also, I do not think that we have any real style rules for imports. I tend to ask for a sorted import list because many modules contain so many imports that it's hard to find out whether something was already imported, but that's just my pet peeve, nothing official.